lua/iswap.lua

```diff
@@ -327,4 +328,14 @@ function M.iswap_with(direction, config)
   vim.cmd([[silent! call repeat#set("\<Plug>ISwapWith", -1)]])
 end
 
+local cmd = vim.api.nvim_create_user_command
+cmd("ISwap", function() require("iswap").iswap() end, {})
```

Comment:
tree-sitter-context does not lua-ize its command definitions. I feel like we should maybe stick to what that does? My reasoning is, if an "official" TS plugin like tree-sitter-context errors on a new installation, then mine probably should too. I'm not sure if that's reasonable though.